### PR TITLE
zeplic 0.1.0

### PR DESCRIPTION
**Pull request by icarbajovallejo**
Process:

1. Get clones dataset
2. Destroy clones dataset
3. Get dataset (called in JSON file)
4. Destroy dataset (disable)
5. Create dataset if it does not exist
6. Save the last #Retain(JSON file) snapshots
7. Create a new snapshot
8. Create a clone of last snapshot
9. Rollback of last snapshot (disable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,53 @@
-zeplic
-======
+# zeplic 0.1.0
```
Comment (rmdir) [inline README.md]: For now no version has been realised

File: README.md [hunk 2]
```diff
@@ -1,6 +1,53 @@
...
+
+## Utils
+
+- System logging daemon.
+
+## How can you use it?
+
+First, clone this repository into `$GOPATH/src/github.com/nfrance-conseil/zeplic` and export your `$GOBIN`.
+The next step is to configure **zeplic**:
+
```
Comment (rmdir) [inline README.md]: you miss go install here. maybe we should issue a Makefile

File: README.md [hunk 3]
```diff
@@ -1,6 +1,53 @@
...
+### Configuration
+
+Add the next line to your syslog configuration file `/etc/syslog.conf`:
+
+```sh
+!zeplic
+local0.*					/var/log/zeplic.d
+```
+
```
Comment (rmdir) [inline README.md]: You should use `local0.* -/var/log/zeplic.d` so syslogd become asynchronous
You should choose the facility by config. Default facility should be daemon

File: README.md [hunk 4]
```diff
@@ -1,6 +1,53 @@
...
+Use a JSON file (/etc/zeplic.d/config.json):
```
Comment (rmdir) [inline README.md]: This works on linux, on FreeBSD, it should be /usr/local/etc/zeplic.d/config.json

File: main.go [hunk 5]
```diff
@@ -1,56 +1,155 @@
 package main
```
Comment (rmdir) [inline main.go:13]: Package comment is missing

File: main.go [hunk 6]
```diff
@@ -1,56 +1,155 @@
...
+// Define the name of the snapshot: SNAP_yyyy-Month-dd_HH:MM:SS
 func snapName() string {
 	year, month, day := time.Now().Date()
 	hour, min, sec := time.Now().Clock()
 	snapDate := fmt.Sprintf("%s_%d-%s-%02d_%02d:%02d:%02d", "SNAP", year, month, day, hour, min, sec)
 	return snapDate
 }
 
+// Get substring with the name of the snapshots
+func between(value string, a string, b string) string {
```
Comment (rmdir) [inline main.go]: This function needs a test

File: README.md (hunk 4 above)
Comment (icarbajovallejo) [reply README.md]: Ok, I going to change the path

File: main.go (hunk 6 above)
Comment (icarbajovallejo) [reply main.go]: You say 'func snapName()'?

File: README.md (hunk 3 above)
Comment (icarbajovallejo) [reply README.md]: In fact, there is an error: /var/log/zeplic.log

File: README.md (hunk 2 above)
Comment (icarbajovallejo) [reply README.md]: Makefile is a solution (including a configuration for the syslogd) or maybe, to do a freebsd port collection with an binary package

File: README.md (hunk 1 above)
Comment (icarbajovallejo) [reply README.md]: Ok, deleted

**Pull request merged**